app/posts/trs-console/2024-01-19-managing-change-requests.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Managing change requests
+date: 2024-01-18
+description: Giving users a way to respond to a teacher’s request to change their name or date of birth in the database.
+screenshots:
+  items:
+    - text: List of change requests
+      src: 1-mq-change-requests-home.png
+
+    - text: Request for a change of first name
+      src: 2-mq-change-request-name.png
+
+    - text: Request for a change of data of birth
+      src: 3-mq-change-request-dob.png
+
+    - text: Confirmation screen shown after a user clicks to accept a change
+      src: 4-mq-change-request-confirm.png
+
+    - text: Reason selection screen shown after a user clicks to reject a change
+      src: 5-mq-change-request-reject-reason.png
+
+    - text: Success screen shows if the change was made and if the user was notified
+      src: 6-mq-change-request-success.png
+---
+
+When teachers [access their teaching qualifications](/qualifications-service) they can change their:
+
+- name
+- date of birth
+- email address
+- mobile phone number
+
+They might do this because something has changed or they notice a mistake.
+
+To change their name or date of birth they need to send a request with proof to support the change. The proof will be photographs or scans of documents such as a birth certificate or passport.
+
+A member of TRA staff needs to check the request and either approve or reject it.
+
+## What we did
+
+We designed a way for TRA staff to deal with requests in the TRS console. It lets users:
+
+- choose a change request from a list
+- view the request including proof
+- accept or reject the request
+
+In the Microsoft Dynamics system which is being replaced, change requests appear in a list of ‘open cases’ for users to review. This is a standard term used in Dynamics.
+
+We’ve decided to use the term ‘change requests’ because it’s a clearer description.
+
+### List of change requests
+
+In the initial design, the list of change requests appears on a user’s ‘home’ tab. We may change this as we add more features to the TRS console.
```

Comment:
I've changed as 'home' will mean different things to different users. I've put 'find a record' as the default, with a nav for 'change requests'.

We could leave this in the entry though as it was true at the time.